cmd/apps/certmanager_app.go

```diff
@@ -87,30 +84,20 @@ func MakeInstallCertManager() *cobra.Command {
 
 		chartPath := path.Join(os.TempDir(), "charts")
 
-		err = helm.FetchChart("jetstack/cert-manager", certManagerVersion, helm3)
-		if err != nil {
-			return err
-		}
-
-		log.Printf("Applying CRD\n")
-		crdsURL := "https://raw.githubusercontent.com/jetstack/cert-manager/release-0.12/deploy/manifests/00-crds.yaml"
-		res, err := k8s.KubectlTask("apply", "--validate=false", "-f",
-			crdsURL)
+		err = helm.FetchChart("jetstack/cert-manager", version, helm3)
 		if err != nil {
 			return err
 		}
 
-		if res.ExitCode > 0 {
-			return fmt.Errorf("error applying CRD from: %s, error: %s", crdsURL, res.Stderr)
-		}
-
 		outputPath := path.Join(chartPath, "cert-manager/rendered")
-		overrides := map[string]string{}
+		overrides := map[string]string{
+			"installCRDs": "true",
```

Comment:
Is this supported for users that may need an older version, or do we need an if statement to apply the CRD via kubectl?

Reply:
This is new since 0.15, older versions need a manual install still. If you want to support them happy to add them using a semver comparison